src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -1100,30 +1097,25 @@ public void onTreeGrow(StructureGrowEvent growEvent)
     @EventHandler(ignoreCancelled = true)
     public void onInventoryPickupItem(InventoryPickupItemEvent event)
     {
-        //prevent hoppers from picking-up items dropped by players on death
-
-        InventoryHolder holder = event.getInventory().getHolder();
-        if (holder instanceof HopperMinecart || holder instanceof Hopper)
+        // Prevent hoppers from taking items dropped by players upon death.
+        if (event.getInventory().getType() != InventoryType.HOPPER)
         {
-            Item item = event.getItem();
-            List<MetadataValue> data = item.getMetadata("GP_ITEMOWNER");
-            //if this is marked as belonging to a player
-            if (data != null && data.size() > 0)
-            {
-                UUID ownerID = (UUID) data.get(0).value();
+            return;
+        }
 
-                //has that player unlocked his drops?
-                OfflinePlayer owner = GriefPrevention.instance.getServer().getOfflinePlayer(ownerID);
-                if (owner.isOnline())
```

Comment:
Kinda weird that he gets an OfflinePlayer when he only cares if person is online, wonder if logic here was different long ago.

Reply:
oh that's weird that it considers this a review, it's just a comment. Anyways the only review is the one above.

Reply:
It might just be habbit - considering it has that `isOnline()` function, that might be more obvious than checking if `getPlayer(UUID)` happens to return a non-null object.

Of course this is without regard to the possible side effect that comes with `Server.getOfflinePlayer(UUID)`.